hummingbird/static/js/hummingbird-ajax.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+$('.delete-user').hide();
+
+$('.delete_device').click(function(){
+    var device;
```

Comment:
nit: Inline var declarations and use single-quotes for selecting and parameters.
```javascript
var device = $(this).attr('data-deviceid');
var user = $(this).attr('data-userprofile');
```